code-of-conduct.md

```diff
@@ -1,3 +1,37 @@
 # OpenTelemetry Community Code of Conduct
 
-OpenTelemetry follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
\ No newline at end of file
+The OpenTelemetry project aims to be a welcoming place where new and existing
+members feel safe to respectfully share their opinions and disagreements. We
+want to attract a diverse group of people to collaborate with us, which means
+acknowledging that people come from different backgrounds and cultures. 
+
+We are bound by the [CNCF Code of
+Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) and
+covered by that in more serious violations and prefer the reporting of
+violations to be made to us as outlined below.
```

Comment:
"covered by that in more serious violations" reads weirdly to me.

```suggestion
This project is bound by and abides by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
Conflicts should be resolved respectfully between the involved parties, however
we understand that it may not always be possible without outside help or mediation.
When needed, any community member can ask for help or report a Code of Conduct
violation to the Governance Committee as outlined below.
The project prefers that violations to be reported first to the Governance Committee
so that they may be resolved internally, however, in extreme cases code of conduct
violations should be reported to the CNCF as outlined in the
[CNCF Incident Resolution Procedures](https://github.com/cncf/foundation/blob/main/code-of-conduct/coc-incident-resolution-procedures.md).
Anyone who is unsure if a violation should be reported to the project or the CNCF should
refer to the [CNCF CoC Jurisdiction Policy](https://github.com/cncf/foundation/blob/main/code-of-conduct/coc-committee-jurisdiction-policy.md).
```